doc/data/messages/c/consider-using-generator/details.rst

```diff
@@ -0,0 +1,6 @@
+Removing ``[ ]`` inside containers should be considered for performance reasons since
```

Comment:
```suggestion
Removing ``[ ]`` inside calls that can use containers or generator indistinctly should be considered for performance reasons since
```